corehq/apps/es/client.py

```diff
@@ -315,7 +316,15 @@ def index_configure_for_standard_ops(self, index):
 
         :param index: ``str`` index for which to change the settings
         """
-        return self._index_put_settings(index, INDEX_CONF_STANDARD)
+        standard_settings = INDEX_CONF_STANDARD | INDEX_SLOWLOG_CONF
+        return self._index_put_settings(index, standard_settings)
+
+    def index_configure_slow_logs(self, index):
```

Comment:
Would it make sense to call in `CreateIndex.run` https://github.com/dimagi/commcare-hq/blob/820073eca0b24a19f67e8331e4b29106c73f126b/corehq/apps/es/migration_operations.py#L83
So that slowlogs config is automatically created after index is created?

Reply:
`manager.index_configure_for_standard_ops` gets called in `CreateIndex.run`, so I was just trying to lump together the index settings changes. Would you prefer these two settings (INDEX_CONF_STANDARD and INDEX_SLOWLOG_CONF) weren't lumped together?